webpack.config.js

```diff
@@ -0,0 +1,69 @@
+var webpack = require('webpack');
+var path = require('path');
+var fs = require('fs');
+
+var externalModules = [
+	'mkfifo',
+	'sqlite3',
+	'mysql2',
+	'pg',
+	'mariasql',
+	'mssql',
+	'mysql',
+	'strong-oracle',
+	'oracle',
+	'oracledb',
+	'pg-query-stream'
+]
+
+// We iterate over the node modules and mark all optional dependencies as external
+var dirs = fs.readdirSync('./node_modules')
+for (let dir of dirs){
+	try {
+		let packageJson = JSON.parse(fs.readFileSync('./node_modules/' + dir + '/package.json'))
+		if (packageJson.optionalDependencies != null){
+			for (let key in packageJson.optionalDependencies) {
+				externalModules.push(new RegExp('^' + key + '(/.*)?$'))
```

Comment:
You could build up an array of optional modules and then turn that into one regex, eg
```
optionalModules = optionalModules.concat(_.keys(packageJson.optionalDependencies))
...
externalModules.push(new RegExp('^' + optionalModules.map(_.escapeRegExp).join('|') + '(/.*)?$'))